spec/models/user_spec.rb

```diff
@@ -1,23 +1,16 @@
 describe User do
-  context 'associations' do
-    it { is_expected.to belong_to(:organisation).optional }
-  end
-
-  context 'validations' do
-    it { is_expected.to validate_presence_of(:name).on(:update) }
-  end
+  it { is_expected.to belong_to(:organisation).optional }
+  it { is_expected.to validate_presence_of(:name).on(:update) }
 
-  context 'permissions' do
-    context 'a new user' do
-      subject { create(:user) }
+  context 'when adding a new user' do
```

Comment:
This context description doesn't really describe the context of the two specs that follow it. Something more descriptive would be `'when creating a user without explicit permissions'`